### PR TITLE
cleaned cabal files

### PR DESCRIPTION
**Pull request by coot**
I just wanted to remove a stale comment:

> Drop dep once use of Data.Semigroup.Action is removed

in `ouroboros-network-framework.cabal` file, but I ended up cleaning it a bit more.

- cleaned ouroboros-netowrk-framework.cabal file
- cleaned ouroboros-network.cabal file

### CODE REVIEW

File: ouroboros-network-framework/ouroboros-network-framework.cabal [hunk 1]
```diff
@@ -60,42 +60,41 @@ library
   build-depends:       base            >=4.12 && <4.13
                      , async           >=2.1  && <2.3
                      , bytestring      >=0.10 && <0.11
+                     , cborg           >=0.2.1 && <0.3
                      , containers      >=0.5  && <0.7
+                     , dns                       < 4.0
+                     , iproute         >= 1.7.0 && < 1.8
```
Comment (mrBliss) [inline ouroboros-network-framework/ouroboros-network-framework.cabal:66]: Alignment :nerd_face:.

File: ouroboros-network-framework/ouroboros-network-framework.cabal [hunk 2]
```diff
@@ -60,42 +60,41 @@ library
...
-                       -Wincomplete-uni-patterns
-                       -Wincomplete-record-updates
-                       -Wpartial-fields
-                       -Widentities
-                       -Wredundant-constraints
+                       -Wincomplete-uni-patterns                            
+                       -Wincomplete-record-updates                          
+                       -Wpartial-fields                                     
+                       -Widentities                                         
+                       -Wredundant-constraints
```
Comment (mrBliss) [inline ouroboros-network-framework/ouroboros-network-framework.cabal:97]: There's a lot of trailing whitespace in this file

File: ouroboros-network-framework/ouroboros-network-framework.cabal (hunk 1 above)
Comment (coot) [reply ouroboros-network-framework/ouroboros-network-framework.cabal:66]: I didn't mind it, but ok let's change it.

File: ouroboros-network-framework/ouroboros-network-framework.cabal (hunk 2 above)
Comment (coot) [reply ouroboros-network-framework/ouroboros-network-framework.cabal:97]: hmm, strange; I'll fix it.

**Comment by coot** [comment]
bors merge

**Comment by nfrisby** [comment]
^^^ (For the record, that "mention" was a typo.)

**Pull request merged**